### DISCUSSION TITLE
Error ssr

### DISCUSSION THREAD

**Discussion by SnoxiK** [inertiajs/inertia#1172] [Help (Vue)]
Hello, I am using laravel 9 with inertia.js.

I configured everything for the SSR as the tutorial recommends, even from scratch I have an error that I didn't have before. Problem I can't debug because when I launch node ssr.js it launches well as in the tutorial but once I try to access the url (which worked a week ago) it exits me an error: 
```
SyntaxError: Unexpected end of JSON input
     at JSON.parse(<anonymous>)
     at /render (/www/mywebsite/node_modules/@inertiajs/server/lib/index.js:32:45)
     at processTicksAndRejections (internal/process/task_queues.js:97:5)
     at async Server.<anonymous> (/www/mywebsite/node_modules/@inertiajs/server/lib/index.js:46:37)
```

Can anyone please help me?

My configuration is exactly the same as the tutorial: https://inertiajs.com/server-side-rendering

Thank you have a good day.

**Comment by MitichPavel** [comment]
Hi, I have the same issue with Laravel, Svelte, Inertia.
It occurs when createInertiaApp try parse variable `$page` and there is no way to debug why it is not valid `JSON`.
`JSON.parse(NOT_VALID_JSON)`
